tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -94,8 +100,7 @@ func TestRateLimiting_DefaultLessThanOverride(t *testing.T) {
 			}
 
 			// check resource exhausted
-			// TODO: until https://github.com/istio/istio/issues/3028 is fixed, use 50% - should be only 5% or so
-			want429s = math.Floor(want429s * 0.50)
+			want429s = math.Floor(want429s * 0.70)
```

Comment:
This will make the condition more stringent. Is that the intent?

Reply:
Yes, I think because we added a warm up before, this should work...